src/views/Timeline.vue

```diff
@@ -143,47 +123,69 @@ export default {
 	},
 
 	methods: {
-		async fetchContent() {
-			// only one simultaneous page load
-			if (this.loadingPage) {
-				return
+		async getContent(params) {
+			if (this.done) {
+				return Promise.resolve([])
 			}
 
 			// cancel any pending requests
 			if (this.cancelRequest) {
 				this.cancelRequest('Changed view')
 			}
 
-			// if we don't already have some cached data let's show a loader
-			if (this.timeline.length === 0) {
-				this.$emit('update:loading', true)
-			}
-			this.error = null
-			this.loadingPage = true
-
-			// init cancellable request
+			// done loading even with errors
 			const { request, cancel } = cancelableRequest(getPhotos)
 			this.cancelRequest = cancel
 
 			try {
-				// get content and current folder info
 				const files = await request(this.onlyFavorites, {
-					page: this.page,
-					perPage: this.gridConfig.count * 5, // we load 5 rows,
+					page: params.offset,
+					perPage: 30,
```

Comment:
Grid config is not used anymore

Reply:
yes, it is useless has the grid is not based on row number anymore, we can decide (or let the user decide) of the number of images to load by batch.

Reply:
reverted